en/controllers/middleware.rst

```diff
@@ -0,0 +1,252 @@
+Middleware
+##########
+
+Middleware give you the ability to 'wrap' your application in re-usable,
+composable layers of Request handling, or response building logic. Middleware
```

Comment:
Is there a plural to Middleware?

http://www.yourdictionary.com/middleware states "Middlewares".

Some use "middlewares", it seems:
https://www.google.ch/#hl=de&q=%22middlewares%22&gws_rd=ssl

For non-English speaker it could be confusing to see this word being used for singular and plural.
So even if "Middlewares" would not be correct, I would recommend it, to use it nonetheless.